apps/src/dance/Dance.js

```diff
@@ -463,7 +463,8 @@ Dance.prototype.shouldShowSharing = function () {
  * This is called while this.p5 is in the preload phase.
  */
 Dance.prototype.onP5Preload = function () {
-  const recordReplayLog = this.shouldShowSharing();
+  //const recordReplayLog = this.shouldShowSharing();
+  const recordReplayLog = experiments.isEnabled('p5Replay');
```

Comment:
This seems like something we don't need to gate on the experiment - my guess is recording the replay log is relatively cheap compared to uploading it - but I bet we _do_ want to only do this on shareable levels, so it's possible your previous implementation here is correct.